puppet.jenkins.io.tf

```
@@ -113,7 +113,7 @@ resource "azurerm_linux_virtual_machine" "puppet_jenkins_io" {
 }
 
 resource "azurerm_dns_a_record" "azure_puppet_jenkins_io" {
-  name                = "azure.puppet.jenkins.io."
+  name                = "azure.puppet" #azure.puppet.jenkins.io --> A
```

```suggestion
  name                = "azure.puppet"
```
Implicit, redundant with the resource type